src/run.ts

```diff
@@ -104,6 +78,44 @@ async function downloadHelm(version: string): Promise<string> {
     return helmpath;
 }
 
+async function getLatestHelmVersionFor(type) {
+    const token = core.getInput('token', { 'required': true });
+    try {
+        const { repository } = await graphql(
+            `{
+      repository(name:"helm", owner:"helm") {
+        releases(last: 100)  {
+            nodes {
+              tagName
+            }
+        }
+      }
+    }`,
+            {
+                headers: {
+                    authorization: `token ${token}`
+                }
+            }
+        );
+
+        const releases = repository.releases.nodes.reverse();
+        let latestValidRelease = releases.find(release => isValidVersion(release.tagName, type));
+        if (latestValidRelease)
+            return latestValidRelease.tagName;
+    } catch (err) {
+        core.warning(util.format("Error while fetching the latest Helm %s release. Error: %s. Using default Helm version %s.", type, err.toString(), stableHelmVersion));
+    }
+    core.warning(util.format("Could not find stable release for Helm %s. Using default Helm version %s.", type, stableHelmVersion));
+    return stableHelmVersion;
```

Comment:
Given that now we support `2.*` and `3.*`, return stable version for the type?

Reply:
Sure. I'll add the most recent Helm 2 and Helm 3 releases as stable versions for each type.